sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -280,6 +280,9 @@ class MapboxMap :
         styleExtension.images.forEach {
           it.bindTo(style)
         }
+        styleExtension.models.forEach {
+          it.bindTo(style)
+        }
```

Comment:
we should also not to forget to add code as discussed in https://github.com/mapbox/mapbox-maps-android/pull/1362/files#r876349084